src/docs/product/issues/states-triage/index.mdx

```diff
@@ -46,6 +46,12 @@ The "Auto Resolve" feature allows you to specify an interval after the last occu
 
 Ignore an issue **permanently** (the default action when you press the button) if it's just noise and you'll likely never care about it. Or ignore it until a condition is met, at which point it becomes Unresolved. You can also ignore an issue if you're working on it and want to silence alerts in the meantime.
 
+<Note>
+  
+Note: Metric Alerts can be triggered by ignored Issues.
```

Comment:
```suggestion
Metric alerts can be triggered by ignored issues.
```

Reply:
Sorry I accidentally pressed merged without adding the suggestion. Will create a new PR to amend.